### PR TITLE
OBSDOCS-478: 1x.extra-small Loki support docs

### PR DESCRIPTION
**Pull request by abrennan89**
Version(s):
4.12+ (Logging 5.8)

Issue:
https://issues.redhat.com/browse/OBSDOCS-478

Link to docs preview:
- https://67435--docspreview.netlify.app/openshift-enterprise/latest/logging/cluster-logging-support#cluster-logging-support-CRDs (moved content, may be outdated)
- https://67435--docspreview.netlify.app/openshift-enterprise/latest/logging/logging_alerts/custom-logging-alerts (removed note)
- https://67435--docspreview.netlify.app/openshift-enterprise/latest/logging/cluster-logging-loki#loki-deployment-sizing_cluster-logging-loki
- https://67435--docspreview.netlify.app/openshift-enterprise/latest/network_observability/installing-operators#loki-deployment-sizing_network_observability (replaced duplicated content with module include)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

File: modules/loki-deployment-sizing.adoc
Comment (abrennan89) [inline modules/loki-deployment-sizing.adoc:15]: @periklis I didn't have any value for this in the Jira so I went with this as a placeholder guess. Please confirm an actual QPS value.

File: logging/cluster-logging-support.adoc
Comment (periklis) [inline logging/cluster-logging-support.adoc:33]:
```suggestion
| RulerConfig    | rulerconfig.loki.grafana/v1    | Supported in 5.7
| AlertingRule   | alertingrule.loki.grafana/v1   | Supported in 5.7
| RecordingRule  | recordingrule.loki.grafana/v1  | Supported in 5.7
```

File: logging/cluster-logging-support.adoc
Comment (periklis) [inline logging/cluster-logging-support.adoc:37]:
```suggestion
```

File: modules/loki-deployment-sizing.adoc
Comment (periklis) [inline modules/loki-deployment-sizing.adoc:19]:
```suggestion
| *Data transfer*              | 100GB/day          | 500GB/day           | 2TB/day
| *Queries per second (QPS)*   | 1-25 QPS at 200ms  | 25-50 QPS at 200ms  | 25-75 QPS at 200ms
| *Replication factor*         | None               | 2                   | 3
| *Total CPU requests*         | 14 vCPUs            | 36 vCPUs          | 54 vCPUs 
| *Total CPU requests incl. Ruler*         | 16 vCPUs            | 42 vCPUs        |  70 vCPUs
| *Total Memory requests*      | 39Gi      | 63Gi              | 109Gi
| *Total Memory requests incl. Ruler*      | 43Gi            | 83Gi            |141Gi
| *Total Disk requests*        | 150Gi              | 300Gi               | 450Gi
```

File: modules/loki-deployment-sizing.adoc
Comment (skrthomas) [inline modules/loki-deployment-sizing.adoc:1]: Can you add // *network_observability/installing-operators.adoc to the inclusion list?

**Review by skrthomas — COMMENTED** [review]
/lgtm thanks @abrennan89 !!

File: modules/loki-deployment-sizing.adoc
Comment (abrennan89) [reply modules/loki-deployment-sizing.adoc:1]: Yes! Sorry I missed it, thank you!

File: modules/loki-deployment-sizing.adoc
Comment (stleerh) [inline modules/loki-deployment-sizing.adoc:22]: Does it need to explain what Ruler is?

If LokiStack is being used for Network Observability, then Ruler isn't being used.

Comment (periklis) [inline modules/loki-deployment-sizing.adoc:22]:
```suggestion
| *Total Disk requests*        | 410Gi              | 410Gi               | 570Gi
| *Total Disk requests incl. Ruler*        | 450Gi              | 450Gi               | 610Gi
```

**Review by periklis — COMMENTED** [review]
I have double checked the numbers one more time, because the original table stems from 5.4/5.5 times and in the meantime we introduced the Work-Ahead Log etc.

File: modules/loki-deployment-sizing.adoc
Comment (abrennan89) [reply modules/loki-deployment-sizing.adoc]: Added conditionals to the module so the ruler part should only show up when this module is included in the Logging docs now, PTAL at the previews 🙂

File: logging/cluster-logging-support.adoc
Comment (jeana-redhat) [inline logging/cluster-logging-support.adoc:34]: Should reformat this with the other style table (example [here](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#alternative-footnote-styling-in-tables), ignore the footnotes bit)

File: logging/cluster-logging-support.adoc
Comment (jeana-redhat) [inline logging/cluster-logging-support.adoc:25]:
```suggestion
LokiStack development is ongoing. Not all APIs are currently supported.
```

File: modules/loki-deployment-sizing.adoc
Comment (jeana-redhat) [inline modules/loki-deployment-sizing.adoc:17]: This one is existing but also should technically be reformatted. I leave this to you, if you have bandwidth to overhaul it now or not.

**Review by jeana-redhat — COMMENTED** [review]
Just a typo and some table formatting stuff, otherwise LGTM

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/loki-deployment-sizing.adoc
Comment (abrennan89) [inline modules/loki-deployment-sizing.adoc:21]:
```suggestion
| *Replication factor*                          | 2                  | 2                   | 2
```

File: modules/loki-deployment-sizing.adoc
Comment (stleerh) [reply modules/loki-deployment-sizing.adoc]: /lgtm

File: modules/loki-deployment-sizing.adoc
Comment (abrennan89) [reply modules/loki-deployment-sizing.adoc]: Great feedback, thank you! For the sake of completeness I've updated both tables.

File: modules/loki-deployment-sizing.adoc
Comment (mjpytlak) [inline modules/loki-deployment-sizing.adoc:14]: While this `ifeval` should work fine, I have never seen one set in the assembly proper. I suggest moving it above the assembly metadata.

File: network_observability/installing-operators.adoc
Comment (mjpytlak) [inline network_observability/installing-operators.adoc:35]: Just want to make sure you intended to offset this module this much. Because of the depth in the assembly, the title does not appear in the mini TOC. [1] IMO it would work perfectly fine with an offset of 2, but totally your call.

[1] https://67435--docspreview.netlify.app/openshift-enterprise/latest/network_observability/installing-operators#loki-deployment-sizing_network_observability

**Review by mjpytlak — COMMENTED** [review]
@abrennan89 Just a few observations ahead of merge.

File: network_observability/installing-operators.adoc
Comment (skrthomas) [reply network_observability/installing-operators.adoc:35]: @mjpytlak This is a great observation. It didn't really jump out at me honestly because in the live docs, this information currently doesn't appear in the TOC due to the fact that its just a table in an assembly. Now that @abrennan89 made it a module it _can_ display in the TOC. I agree with making it a level 2 so it will be displayed. I think sometimes customers are just looking for this reference and it would be nice to show in the mini TOC.

**Comment by abrennan89** [comment]
Thanks @mjpytlak ! Should be good to go now 🤞🏻 If you want to merge it that's fine, or you can just mark as approved and I can do the merge and CPs myself.

**Review by mjpytlak — APPROVED** [review]

**Pull request merged**
